### ISSUE TITLE
Partial Expressions in require call

### ISSUE THREAD

**Issue by alanbentley** [webpack-contrib/karma-webpack#289]
**I'm submitting a bug report**

**Webpack version:**
3.10.0

**Webpack Karma version**:
2.0.8/9

**Karma version**:
0.12.37

**Please tell us about your environment:**
OSX 10.12.6

**Browser:** 
Phantom

**Current behavior:**
partial expressions in require statements were working fine in version 2.0.5-7 but are now causing 404 responses

**Expected/desired behavior:**
Asynchronously load module

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem along with a gist/jsbin of your webpack configuration.**

`require(['views/' + view], function(view) {`

works fine in Use karma-webpack 2.0.7 however using karma-webpack 2.0.8-2.0.9:

`WARN [web-server]: 404: /var/folders/qr/vvd9_2fx45d5pfgfbb8qhbmh0000gn/T/_karma_webpack_/0.bundle.js`

Changing the dependancy to a hard coded string enables the module to be found.

**Comment by alanbentley** [comment]
Had a quick look and looks like the issue is in [0616dda](https://github.com/webpack-contrib/karma-webpack/commit/0616dda489d07f6a9bc3def1778dcef601411821) with the new regex:

`urlRegex: new RegExp('^' + os.tmpdir() + '\/_karma_webpack_\/.*/')`

a quick test by changing this to the following resolves finding the module:

`urlRegex: /.*\/_karma_webpack_\/.*/,`

**Comment by andrexx** [comment]
Apparently broken the possibility of loading lazy load bundles. Got a similar issue on windows with angular application. As workaround pin karma-webpack version to 2.0.6
```
Chrome 63.0.3239 (Windows 10 0.0.0): Executed 73 of 318 SUCCESS (0 secs / 0.109 secs)
ERROR: 'Unhandled Promise rejection:', 'Loading chunk 0 failed.', '; Zone:', 'ProxyZone', '; Task:', 'Promise.then', ';
Value:', Error{__zone_symbol__currentTask: ZoneTask{_zone: Zone{_properties: ..., _parent: ..., _name: ..., _zoneDelegate: ...}, runCount: 0,
_zoneDelegates: null, _state: 'notScheduled', type: 'microTask', source: 'Promise.then',
data: undefined, scheduleFn: undefined, cancelFn: null, callback: function () { ... }, invoke: function () { ... }}}, 'Error: Loading chunk 0 failed.
    at HTMLScriptElement.onScriptComplete 
...
```

**Comment by rodmax** [comment]
Hi guys could you verify fix from #293 
Quick steps to setup env
```
git clone git@github.com:rodmax/karma-webpack.git
cd karma-webpack
git checkout fix-url-regex-for-custom-file-handler
npm i 
npm link
npm i webpack@3.6.0   # you should install peer dependency for using "npm link", you may choose your version of webpack instead 3.6.0
npm run build  # after it you should see "js" files in ./lib dir

cd path/to/you-repo
npm link karma-webpack  # mount my pull-requested version
npm run tests # run your tests
```

**Comment by alanbentley** [comment]
Sorry for the delay - I can confirm that fix-url-regex-for-custom-file-handler fixes my issue

**Comment by andrexx** [comment]
Unfortunately I keep getting an error "Error: Loading chunk 0 failed.".
I'll try to create a minimal reproducible example.

**Comment by michael-ciniawsky** [comment]
Fix released in [`v2.0.10`](https://github.com/webpack-contrib/karma-webpack/releases/tag/v2.0.10)  🎉  Please try it out. If still regressions please open a new issue :)
